espnet2/bin/whisper_export_vocabulary.py

```diff
@@ -59,9 +66,14 @@ def export_vocabulary(output: str, whisper_model: str, language: str, log_level:
     # NOTE (Shih-Lun): extra tokens (for timestamped ASR) not
     #                  stored in the wrapped tokenizer
     full_vocab_size = 51865 if whisper_model == "whisper_multilingual" else 51864
+
     for i in range(full_vocab_size - vocab_size):
         fout.write("()" + "\n")
 
+    if sot_asr:
```

Comment:
The exported `token.txt` file includes 50257 normal tokens, 1501 "{}" (should be timestamps), and 1 `sc` token.
During training, the tokenizer will add additional 1501 timestamps tokens and 1 sc token, as implemented in `whisper_token_id_converter.py` and `whisper_token_id_converter.py`.
Although the `token.txt` file will not be used in practice, it is better to change the `{}` to real timestamps and make it consistent with training progress.

Reply:
Thanks for your comments, I also feel like that will be better.  Now updated.